File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/interactor/error/UpdateNumberValidationError.java
package com.lattis.ellipse.domain.interactor.error;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by raverat on 2/20/17.
 */

public class UpdateNumberValidationError extends Throwable {

    private List<Status> status = new ArrayList<>();

    public UpdateNumberValidationError(List<Status> status) {
        this.status = status;
    }

    public List<Status> getStatus() {
        return status;
    }

    public enum Status {
        INVALID_PHONE_NUMBER
    }

}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/usecase/error/UpdateNumberValidationError.kt
package com.lattis.domain.usecase.error

import java.util.*

class UpdateNumberValidationError(status: List<Status>) :
    Throwable() {
    var status: List<Status> =
        ArrayList()

    enum class Status {
        INVALID_PHONE_NUMBER
    }

    init {
        this.status = status
    }
}